### DISCUSSION TITLE
Logger timestamp

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [AliMD/alwatr#76] [Q&A] [answered]
What do you think a feature should be added to Logger?
Feature that optionally also shows the log time

**Comment by alimd** [comment, marked as answer]
Press `ctrl+shift+p` and type `time`! :trollface: 

<img width="497" alt="Screen Shot 2022-03-11 at 5 09 02 AM" src="https://user-images.githubusercontent.com/697913/157785436-bd3efa92-3a6d-4f54-ac24-57776b2eb4dc.png">

**Comment by njfamirm** [comment]
For node?
